### PR TITLE
Issue66 send

### PR DESCRIPTION
**Pull request by Lrthayer**
adds picking user functionality to messaging.

### CODE REVIEW

File: public_html/dashboard.js [hunk 1]
```diff
@@ -56,8 +56,16 @@ $(document).ready(function () {
 
   // Event Listener for send message button
   $('#send').on ('click', function () {
-    $.post ('/dashboard/compose/send', {}, function (message) {
-      console.log (message);
+	  var messageText = $('#messageText').val();
+	  var recipient = $('#userRecipient').val();
+	  var messageTitle = $('#messageTitle').val();
+	  var data = {
+		  val: messageText,
```
Comment (DannyPeck) [inline public_html/dashboard.js]: I would probably change this from val to something like content or message

File: app/controllers/DashboardController.js [hunk 2]
```diff
@@ -51,17 +51,15 @@ DashboardController.prototype.messages = function () {
...
     return function (req, res) {
-
         var token = req.user;
-        var contact = 'danieljpeck93@gmail.com';
+        var contact = req.body.user;
         var dt = Date.now()+10000;
-        var title = "Demo Day!!!";
-        var message = "Hey that was a pretty cool demo, right??";
+        var title = req.body.title;
+        var message = req.body.val;
```
Comment (DannyPeck) [inline app/controllers/DashboardController.js]: I would probably change this from val to something like content or message

**Review by DannyPeck — CHANGES_REQUESTED** [review]

File: public_html/dashboard.js (hunk 1 above)
Comment (Lrthayer) [reply public_html/dashboard.js]: agreed, content it is.

**Review by DannyPeck — APPROVED** [review]

**Review by timwhitf — APPROVED** [review]
Looks good I think.

**Pull request merged**